### PR TITLE
[16.0][MIG] sale_partner_selectable_option: Migration to version 16.0

### PR DESCRIPTION
**Pull request by Yadier-Tecnativa**
@Tecnativa  TT42434

### CODE REVIEW

**Comment by Yadier-Tecnativa** [comment]
@sergio-teruel  ping, and @pedrobaeza , @victoralmau

**Review by victoralmau — APPROVED** [review]
Code review OK

File: sale_partner_selectable_option/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from lxml import etree
+
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.model
+    def fields_view_get(
```
Comment (pedrobaeza) [inline sale_partner_selectable_option/models/sale_order.py]: This is deprecated:

https://github.com/OCA/sale-workflow/actions/runs/4822025773/jobs/8588705236?pr=2495#step:8:117

Comment (Yadier-Tecnativa) [reply sale_partner_selectable_option/models/sale_order.py]: Changed using as reference:  https://github.com/odoo/odoo/commit/b03c227e885efa4ccdcad43ccb56ee10371b9284#

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Comment by sergio-teruel** [comment]
ping @juangomezzurita

**Pull request merged**